docker/dev/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Engelsystem PHP FPM development image including Xdebug
+FROM php:7.3-fpm-alpine3.10 AS php_fpm
+WORKDIR /var/www
+RUN apk add --no-cache icu-dev $PHPIZE_DEPS && \
+    pecl install xdebug && \
+    docker-php-ext-install intl pdo_mysql && \
+    docker-php-ext-enable xdebug
+COPY ./docker/dev/php/xdebug.ini /usr/local/etc/php/conf.d
+
+ENV TRUSTED_PROXIES 10.0.0.0/8,::ffff:10.0.0.0/8,\
+                    127.0.0.0/8,::ffff:127.0.0.0/8,\
+                    172.16.0.0/12,::ffff:172.16.0.0/12,\
+                    192.168.0.0/16,::ffff:192.168.0.0/16,\
+                    ::1/128,fc00::/7,fec0::/10
+
+# Engelsystem development workspace
+# Contains all tools required to build / manage the system
+FROM php_fpm AS workspace
+RUN apk add --no-cache composer gettext nodejs npm yarn
```

Comment:
Hm, im not shure if thats the best way as it means that it would use another version that the one in `docker/nginx/Dockerfile`

Reply:
Using the same versions as in the prod Docker setup